src/domain/users/index.types.d.ts

```diff
@@ -31,7 +31,9 @@ type UserQuizQuestion = {
 
 type User = {
   readonly id: UserId
-  readonly contacts: WalletContact[]
+  readonly username: Username
+  readonly walletPublicId: WalletPublicId
```

Comment:
This starts to get a little confusing for me. My current understanding is that an `Account` can have a default `WalletPublicId` and a `User` can have a default `AccountId` that can be used to get to a default `WalletPublicId`.